src/components/redemption/Confirming.js

```diff
@@ -1,68 +1,55 @@
-import React, { Component } from 'react'
+import React from 'react'
 import { connect } from 'react-redux'
 
 import StatusIndicator from '../svgs/StatusIndicator'
 
-class Confirming extends Component {
-  handleClickButton = () => {
-    const { txHash, btcNetwork } = this.props
+const Confirming = ({ txHash, btcNetwork, error }) => {
+  const blockExplorerUrl =
+    `https://blockstream.info/${btcNetwork === 'testnet' ? 'testnet/' : ''}tx/${txHash}`
 
-    window.open(
-      `https://blockstream.info/${btcNetwork === 'testnet' ? 'testnet/' : ''}tx/${txHash}`,
-      '_blank'
-    );
-  }
-
-  render() {
-    const {
-      error,
-      txHash
-    } = this.props
-
-    return (
-      <div className="confirming">
-        <div className="page-top">
-          <StatusIndicator pulse />
+  return (
+    <div className="confirming">
+      <div className="page-top">
+        <StatusIndicator pulse />
+      </div>
+      <div className="page-body">
+        <div className="step">
+          Step 4/6
+        </div>
+        <div className="title">
+          { error ? 'Error confirming transaction' : 'Confirming...' }
         </div>
-        <div className="page-body">
-          <div className="step">
-            Step 4/6
-          </div>
-          <div className="title">
-            { error ? 'Error confirming transaction' : 'Confirming...' }
-          </div>
-          <hr />
-          <div className="description">
-            <p>We're waiting to confirm your transaction.</p>
-            {
-              txHash
-              ? <button
-                  onClick={this.handleClickButton}
-                  className="black"
-                  >
+        <hr />
+        <div className="description">
+          <p>We're waiting to confirm your transaction.</p>
+          {
+            txHash
+            ? <button className="black">
```

Comment:
Again, no real need for a `button` here.

Reply:
Addressed in d820efb